os/syscall/syscall_stublookup.c

```diff
@@ -377,6 +378,10 @@ uintptr_t STUB_prctl(int nbr, uintptr_t parm1, uintptr_t parm2,
 
 uintptr_t STUB_fin_wait(int nbr);
 
+int STUB_notsupport_int(void);
+void *STUB_notsupport_ptr(void);
+void STUB_notsupport_void(void);
+
```

Comment:
We dont need to add these as we are already including syscall_not_support.h here.